docs/_functions/domain/CAA.md

```diff
@@ -0,0 +1,34 @@
+---
+name: CAA
+parameters:
+  - name
+  - tag
+  - value
+  - modifiers...
+---
+
+CAA adds a CAA record to a domain. The name should be the relative label for the record. Use `@` for the domain apex.
+
+Tag can be one of "issue", "issuewild" or "iodef". Value depends on the tag, and is automatically quoted.
+
```

Comment:
What about "target"?   Is it like NS or CNAME where if it doesn't end in a "." the domain will be added?  I assume not, but it would be good to note.